packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -349,13 +351,41 @@ class RenderUiKitView extends RenderBox {
 
   @override
   void handleEvent(PointerEvent event, HitTestEntry entry) {
-    if (event is PointerDownEvent) {
-      _gestureRecognizer.addPointer(event);
+    if (!(event is PointerDownEvent)) {
```

Comment:
nit: `if (event is! PointerDownEvent) {`

Reply:
done